packages/teams-js/src/public/pages.ts

```diff
@@ -1007,11 +1006,13 @@ export namespace pages {
   }
 
   /**
-   * Provides functions for navigating without needing to specify your application ID.
+   * Provides functions for navigating within your own app
```

Comment:
Given doc rendering, you might consider not having the blank line or having the second line be a `@remark`.